Kconfig.zephyr

```diff
@@ -143,24 +143,6 @@ config CUSTOM_LINKER_SCRIPT
 	  linker script and avoid having to change the script provided by
 	  Zephyr.
 
-config CUSTOM_RODATA_LD
-	bool "Include custom-rodata.ld"
-	help
-	  Include a customized linker script fragment for inserting additional
-	  data and linker directives into the rodata section.
-
-config CUSTOM_RWDATA_LD
-	bool "Include custom-rwdata.ld"
-	help
-	  Include a customized linker script fragment for inserting additional
-	  data and linker directives into the data section.
-
-config CUSTOM_SECTIONS_LD
-	bool "Include custom-sections.ld"
-	help
-	  Include a customized linker script fragment for inserting additional
-	  arbitrary sections.
-
```

Comment:
Before dropping support for CUSTOM_RODATA_LD we might want to first mark it as deprecated to give users a reasonable window of time to migrate.